pandas/tests/io/json/test_pandas.py

```diff
@@ -1032,6 +1038,65 @@ def test_to_jsonl(self):
         assert result == expected
         assert_frame_equal(pd.read_json(result, lines=True), df)
 
+    def test_readjson_chunks(self, lines_json_df):
```

Comment:
can you add a test which writes a file in line-format and reads it back. check that the file handle is closed properly.

Reply:
done